src/content/en/updates/2017/04/headless-chrome.md

```diff
@@ -335,13 +343,40 @@ Demos
 
 ## FAQ
 
+**Do I need the `--disable-gpu` flag?**
+
+Yes, for now.  The `--disable-gpu` flag is a temporary requirement to work around a few bugs. You won't need this
+flag in future versions of Chrome. See [https://crbug.com/546953#c152](https://bugs.chromium.org/p/chromium/issues/detail?id=546953#c152) and [https://crbug.com/695212](https://bugs.chromium.org/p/chromium/issues/detail?id=695212) for more information.
+
+**So I still need Xvfb?**
+
+Nope. Headless Chrome doesn't use a window so a display server like Xvfb is
+no longer needed. You can happily run your automated tests without it.
+
+What is Xvfb? Xvfb is an in-memory display server for Unix-like systems that enables you
+to run graphical applications (like Chrome) without an attached physical display.
+Many people use Xvfb to run earlier versions of Chrome to do "headless" testing.
+
 **How do I create a Docker container that runs Headless Chrome?**
 
 Check out [lighthouse-ci](https://github.com/ebidel/lighthouse-ci). It has an
 [example Dockerfile](https://github.com/ebidel/lighthouse-ci/blob/master/builder/Dockerfile)
 that uses Ubuntu as a base image, and installs + runs Lighthouse in an App Engine
 Flexible container.
 
+**Can I use this with Selenium / WebDriver / ChromeDriver**?
+
+Right now, Selenium opens a full instance of Chrome. In other words, it's
+automated solution but not completely headless. However, Selenium could use
+`--headless` in the future.
+
+If you want to bleed on the edge, I recommend [Running Selenium with Headless Chrome](https://intoli.com/blog/running-selenium-with-headless-chrome/) to set things up
```

Comment:
The expression is "bleeding edge". You've altered it slightly, which is a subtlety that might get missed by a translator.